Odata-docs/client/v6/batch-operations.md

```diff
@@ -164,3 +164,61 @@ The request Payload is as following:
 	--changeset_b98a784d-af07-4723-9d5c-4722801f4c4d--
 	--batch_06d8a02a-854a-4a21-8e5c-f737bbd2dea8--
 ```
+
+## Relative Uris in Batch Requests
+`ODataServiceContext` provides `SaveChangesOptions.UseRelativeUri` which allows individual requests in batch to use Relative Uris. In this case, they will rely on the Base Uri of the top level request.
+
+### Relative Uris in Batch Queries
+We pass `SaveChangesOptions.BatchWithSingleChangeset` or `SaveChangesOptions.BatchWithIndependentOperations` with `SaveChangesOptions.UseRelativeUri` in the ExecuteBatch and ExecuteBatchAsync Methods af follows.
+
+```
+var batchResponse = dsc.ExecuteBatch(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri, peopleQuery, airlinesQuery);
+```
+```
+var batchResponse = await dsc.ExecuteBatchAsync(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri, peopleQuery, airlinesQuery);
+```
+
+### Relative Uris in Batch Modification
+We pass `SaveChangesOptions.BatchWithSingleChangeset` or `SaveChangesOptions.BatchWithIndependentOperations` with `SaveChangesOptions.UseRelativeUri` in the SaveChanges and SaveChangesAsync Methods af follows.
+
+```
+dsc.SaveChanges(SaveChangesOptions.BatchWithSingleChangeset | SaveChangesOptions.UseRelativeUri);
+```
+```
+await dsc.SaveChangesAsync(SaveChangesOptions.BatchWithSingleChangeset | SaveChangesOptions.UseRelativeUri);
+```
+
+A truncated request Payload will look like this:
+
+```html
+
+	--batch_06d8a02a-854a-4a21-8e5c-f737bbd2dea8
+	Content-Type: multipart/mixed; boundary=changeset_b98a784d-af07-4723-9d5c-4722801f4c4d
+	.....
+	.....
+	.....
+
+	GET /People HTTP/1.1
```

Comment:
```suggestion
	GET /People HTTP/1.1  // Relative URI is used
```
A short comment here so people know what they are looking at and what to look out for